public/javascripts/data.js

```diff
@@ -148,6 +148,23 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
         'Vertical Slider': 'vertical',
         'Picker': 'picker'
     };
+    var mediaTypes = {
```

Comment:
If we are adding media types, the `new` appearance is something folks want. 
https://docs.opendatakit.org/form-question-types/#image-widget-with-required-new-image

Reply:
okay